tests/web/test_server.py

```diff
@@ -299,3 +301,72 @@ def test_server_timing_header(
 
 def test_exact_showcase_output(showcase_output: str) -> None:
     assert SHOWCASE_RESUME == showcase_output
+
+
+@pytest.mark.filterwarnings("ignore:Request.message is deprecated")
+@pytest.mark.filterwarnings("ignore:Exception ignored in")
+class TestWebHandler:
+    async def test_web_handler_basic_functionality(
+        self,
+        aiohttp_client: Any,
+        aiohttp_server: Any,
+    ) -> None:
+        # Set up a mock resume server
+        async def mock_resume_handler(request):
+            return Response(
+                text=json.dumps({"basics": {"name": "Test User", "label": "Developer"}})
```

Comment:
You could use: https://docs.aiohttp.org/en/stable/_modules/aiohttp/web_response.html#json_response

Reply:
Done following the suggestion, can you please take a look?